hackathon_site/hardware/serializers.py

```diff
@@ -0,0 +1,41 @@
+from rest_framework import serializers
+from hardware.models import Hardware, Category, Order
+
+
+class HardwareSerializer(serializers.ModelSerializer):
+    quantity_remaining = serializers.SerializerMethodField()
+
+    class Meta:
+        model = Hardware
+        fields = (
+            "id",
+            "name",
+            "model_number",
+            "manufacturer",
+            "datasheet",
+            "quantity_available",
+            "notes",
+            "max_per_team",
+            "picture",
```

Comment:
This guy is a little bit weird coming from the fixture. For the value `"/home/pictures"`, the API returns `"http://localhost:8000/api/hardware/hardware/home/pictures"`. If this is something to fix another time (which it probably is, since we need to configure file uploads), we should call it out in a ticket.

Reply:
Created new issue.